Tests/ArmoniK.EndToEndTests/ArmoniK.EndToEndTests.Client/Tests/CheckGridServer/SimpleGridServerClientTest.cs

```diff
@@ -0,0 +1,144 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using ArmoniK.DevelopmentKit.Common;
+
+using NUnit.Framework;
+
+namespace ArmoniK.EndToEndTests.Client.Tests.CheckGridServer;
+
+public class SimpleGridServerClientTest
+{
+  private const string ApplicationNamespace = "ArmoniK.EndToEndTests.Worker.Tests.CheckGridServer";
+  private const string ApplicationService   = "SimpleGridServerTestWorker";
+
+  private readonly double[] numbers_ = new List<double>
```

Comment:
This could be made static. It can also be Generated via LinQ:
```suggestion
  private static readonly double[] numbers_ = Enumerable.Range(0, 10).Select(i => (double)i).ToArray();
```